File: Java/TestingSystem_Assignment4/src/com/vti/entity/Account.java
package com.vti.entity;

import java.time.LocalDate;

public class Account {
	private int id;
	private String email;
	private String userName;
	private String fullName;
	public Department department;
	public Position position;
	public LocalDate createDate;
	public Account() {
		super();
	}

	public Account(int id, String email, String userName, String firstName, String lastName) {
		super();
		this.id = id;
		this.email = email;
		this.userName = userName;
		this.fullName = firstName + lastName;
	}

	public Account(int id, String email, String userName, String firstName, String lastName, Position position) {
		super();
		this.id = id;
		this.email = email;
		this.userName = userName;
		this.fullName = firstName + lastName;
		this.position = position;
		this.createDate = LocalDate.now();
	}

	public Account(int id, String email, String userName, String firstName, String lastName,Position position,
			LocalDate createDate) {
		super();
		this.id = id;
		this.email = email;
		this.userName = userName;
		this.fullName = fullName;
		this.department = department;
		this.position = position;
		this.createDate = createDate;
	}

	
	public Account(int id, String email, String userName, String fullName) {
		super();
		this.id = id;
		this.email = email;
		this.userName = userName;
		this.fullName = fullName;
	}

	public Account(int id, String email, String userName, String fullName, Position position) {
		super();
		this.id = id;
		this.email = email;
		this.userName = userName;
		this.fullName = fullName;
		this.position = position;
	}

	

	public Account(int id, String email, String userName, String fullName, Position position, LocalDate createDate) {
		super();
		this.id = id;
		this.email = email;
		this.userName = userName;
		this.fullName = fullName;
		this.position = position;
		this.createDate = createDate;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getUserName() {
		return userName;
	}

	public void setUserName(String userName) {
		this.userName = userName;
	}

	public String getFullName() {
		return fullName;
	}

	public void setFullName(String fullName) {
		this.fullName = fullName;
	}

	public Department getDepartment() {
		return department;
	}

	public void setDepartment(Department department) {
		this.department = department;
	}

	public Position getPosition() {
		return position;
	}

	public void setPosition(Position position) {
		this.position = position;
	}

	public LocalDate getCreateDate() {
		return createDate;
	}

	public void setCreateDate(LocalDate createDate) {
		this.createDate = createDate;
	}

}

	

File: ReactJs/Lab1/lLab2_TypeScriptOOp/Account.js
"use strict";
exports.__esModule = true;
exports.printIn4Account = exports.Account = void 0;
var Account = /** @class */ (function () {
    function Account(id, email, username, fullname, department, position, createDate) {
        this.id = id;
        this.email = email;
        this.username = username;
        this.fullname = fullname;
        this.department = department;
        this.position = position;
        this.createDate = createDate;
    }
    /**
     * Getterid
     *
     */
    Account.prototype.getid = function () {
        return this.id;
    };
    /**
     * Getteremail
     *
     */
    Account.prototype.getemail = function () {
        return this.email;
    };
    /**
     * Getterusername
     *
     */
    Account.prototype.getusername = function () {
        return this.username;
    };
    /**
     * Getterfullname
     
     */
    Account.prototype.getfullname = function () {
        return this.fullname;
    };
    /**
     * Getterdepartment
     * }
     */
    Account.prototype.getdepartment = function () {
        return this.department;
    };
    /**
     * Getterposition
     *
     */
    Account.prototype.getposition = function () {
        return this.position;
    };
    /**
     * GettercreateDate
     *
     */
    Account.prototype.getcreateDate = function () {
        return this.createDate;
    };
    /**
     * Setterid
     *
     */
    Account.prototype.setid = function (value) {
        this.id = value;
    };
    /**
     * Setteremail
     *
     */
    Account.prototype.setemail = function (value) {
        this.email = value;
    };
    /**
     * Setterusername
     *
     */
    Account.prototype.setusername = function (value) {
        this.username = value;
    };
    /**
     * Setterfullname
     *
     */
    Account.prototype.setfullname = function (value) {
        this.fullname = value;
    };
    /**
     * Setterdepartment
     *
     */
    Account.prototype.setdepartment = function (value) {
        this.department = value;
    };
    /**
     * Setterposition
     *
     */
    Account.prototype.setposition = function (value) {
        this.position = value;
    };
    /**
     * SettercreateDate
     *
     */
    Account.prototype.setcreateDate = function (value) {
        this.createDate = value;
    };
    return Account;
}());
exports.Account = Account;
function printIn4Account(account) {
    console.log("Id: " + account.getid()
        + "email: " + account.getemail()
        + "username: " + account.getusername()
        + "fullname: " + account.getfullname()
        + "department: " + account.getdepartment().getName
        + "position: " + account.getposition().getname
        + "createDate: " + account.getcreateDate());
}
exports.printIn4Account = printIn4Account;
